Tests/LiquidAwsS3DriverTests/LiquidAwsS3DriverTests.swift

```diff
@@ -5,13 +5,14 @@ final class LiquidAwsS3DriverTests: XCTestCase {
     
     let key = "****"
     let secret = "****"
-    let bucket = "****"
+    let bucket = "bucket"
```

Comment:
I think mocking would be best, especially since some tests rely on the custom endpoint while others rely on a non custom endpoint... This would ultimately require anyone running tests to have an account both with AWS and a third party.

That said, I'm still wrapping my head around futures and promises so I'm not sure this is something I could contribute. I'll definitely noodle on it though.